src/main/resources/portal.properties.EXAMPLE

```diff
@@ -71,7 +71,8 @@ skin.study_view.link_text=To build your own case set, try out our enhanced Study
 authorization=false
 ## enable and set this property to specify a study group to be used to identify public studies for which no specific authorization entries are needed in the `authorities` table
 # always_show_study_group=
-## which method of authentication to use (false, googleplus, saml, openid, ad, ldap, noauthsessionservice)
+## which method of authentication to use (false, public_auth, googleplus, saml, openid, ad, ldap, noauthsessionservice)
```

Comment:
Isn't this supposed to be "social_auth"?  If so, please also change the PR description to be "social_auth".